mache-chroniclemap/src/test/java/com/excelian/mache/chroniclemap/ChronicleMapMacheProvisionerShould.java

```diff
@@ -0,0 +1,88 @@
+package com.excelian.mache.chroniclemap;
+
+import com.excelian.mache.chroniclemap.ChronicleMapMacheProvisioner.ChronicleMapBuilderConfig;
+import com.excelian.mache.core.HashMapCacheLoader;
+import net.openhft.chronicle.map.ChronicleMapBuilder;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.util.List;
+
+import static com.excelian.mache.chroniclemap.ChronicleMapMacheProvisioner.chronicleMap;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.internal.util.reflection.Whitebox.getInternalState;
+
+public class ChronicleMapMacheProvisionerShould {
+
```

Comment:
this is a Sandro Mancuso special - and I like it too as in intellij, if you run the tests the list bottom left reads really well